### ISSUE TITLE
Specify licenses.

### ISSUE THREAD

**Issue by ngokevin** [aframevr/assets#1]
In each folder, we can have a `LICENSE` file that lists the license for each asset.

**Comment by cvan** [comment]
Creative Commons Public Domain seem kosher? (May want to ask Legal too. And if we're using assets under different licenses, let's figure that out or remove it.)

**Comment by cvan** [comment]
We should also add a root-level `LICENSE.md` file.

**Comment by leweaver** [comment]
Adding a root level license (CC or MIT preferably) would allow us to contribute to this repository - can this be done?

**Comment by dmarcos** [comment]
Absolutely. I opened a PR to include the License. The rest of our repos fetch and use files from this repo and they're all under MIT license. CC would also work but I think it's better to keep it simple and consistent.

**Comment by dmarcos** [comment]
@leweaver Probably the simpler option is to follow the `THREE` model of including the license in the directory with the assets as @fernandojsg described (https://github.com/aframevr/assets/pull/28#issuecomment-327945058). Can you create a PR with the models in a new directory `Microsoft` in https://github.com/aframevr/assets/tree/gh-pages/controllers and include the license there (CC or MIT)? Would that work?

**Comment by leweaver** [comment]
Just checking with Legal here on taking that approach, will let you know!

**Comment by leweaver** [comment]
That approach works for us, we'll create a PR with MIT license within a Microsoft subfolder.
